### PR TITLE
Updating for heading changes in cf-core

### PR DESCRIPTION
**Pull request by jimmynotjim**
Updated the atomic pieces for the heading changes in cf-core

## Removals

- Completely removed text-intro stylesheet in favor of default styling.

## Changes

- Updated atoms to use default styling where possible.
- Updated visually adjusted spacing to use full values (we're no longer visually adjusting spacing).
- Broke up text-intro molecule params to settings and options objects for clarity on what is and isn't required.
- General code organization cleanup.

## Testing

- Build the project and browse the prototype pages. Headings should use default margins, size, and line-heights. Headings outside of text-intro molecule and full-width molecule should not include margin-top.

## Review

- @KimberlyMunoz 
- @sebworks
- @anselmbradford 

## Notes

- Does not include updates to the existing About section code. That should be replaced with atomic parts later.

## Checklist

* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices 
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

File: cfgov/jinja2/v1/_includes/molecules/text-introduction.html [hunk 1]
```diff
@@ -1,57 +1,51 @@
...
 
-   settings.heading:   Heading text.
-   settings.intro:     Body introduction text.
-   settings.body:      Body text.
+   settings.heading:  Heading text.
+   settings.body:     Body text.
 
-   settings.link_url:  Link URL.
-   settings.link_text: Link text.
+   options:           An object with the following values for options.
```
Comment (KimberlyMunoz) [inline cfgov/jinja2/v1/_includes/molecules/text-introduction.html]: The difference between settings and options is optionality, right?

Comment (jimmynotjim) [inline cfgov/jinja2/v1/_includes/molecules/text-introduction.html]: Yep, just like I did on the hero. We don't have to organize them this way if everyone else disagrees, but it seems to be the most consistent to me.

**Comment by KimberlyMunoz** [comment]
This spacing seems a little off. Is it correct?

image

**Comment by jimmynotjim** [comment]
@KimberlyMunoz sorry, I should have specified that there's an existing issue with `.date` in cf-typography that's outside the spec of this PR. There's already a PR to fix that in cf-typography.

**Comment by KimberlyMunoz** [comment]
Cool. I'm trying to just give it a quick test with the stuff on `/landing-page/`

**Comment by KimberlyMunoz** [comment]
`/landing-page/` looks good. :+1: 

Let me know if this needs another review when some of the CF gets merged in.

**Comment by sebworks** [comment]
:+1:

**Comment by jimmynotjim** [comment]
@KimberlyMunoz @sebworks I just merged and released the related changes in CF components. Run `bower update && gulp build` if you'd like to test this locally with those changes.

**Comment by sebworks** [comment]
:+1:

**Comment by KimberlyMunoz** [comment]
:+1:

File: cfgov/jinja2/v1/_includes/molecules/image-text-25-75.html [hunk 2]
```diff
@@ -49,53 +51,58 @@
...
             {% endif %}
         </div>
-        <div class="m-image-text-25-75_content">
-            <h3 class="m-image-text-25-75_heading">{{ value.heading }}</h3>
-
-            <p class="m-image-text-25-75_body">{{ value.body | safe }}</p>
 
+        <div class="m-image-text-25-75_content">
+            <h3>{{ value.heading }}</h3>
+            <p>{{ value.body | safe }}</p>
```
Comment (kurtw) [inline cfgov/jinja2/v1/_includes/molecules/image-text-25-75.html]: Will this need to have markup in it? If so, are the `p` tags necessary?

Comment (jimmynotjim) [inline cfgov/jinja2/v1/_includes/molecules/image-text-25-75.html]: I am not really sure, it's the same as before, I just removed the extra line break between them while cleaning things up a bit.

**Pull request merged**